xml/vm_security.xml

```diff
@@ -0,0 +1,202 @@
+
+    <para>
+      Support for AMD SEV-SNP is available as a Technology Preview in SUSE Linux Enterprise Server 15-SP6. However, the necessary packages are not part of the default installation or repositories.</para>
+    <para>The packages are shipped via Confidential Compute Module. You must enable it at system installation time or later via the SUSEConnect command line tool.</para>
+    <itemizedlist>
+      <listitem>
+        <para>To check whether the module is already enabled, run the command:
+        </para>
+        <screen># suseconnect -l</screen>
+        <para>This displays the list of available modules with their activation status and commands to enable the inactive modules.</para>
+      
+      <para>The inactive confidential compute module appears as given below:</para>
+    <screen>Confidential Computing Technical Preview Module 15 SP6 x86_64
+      Activate with: suseconnect -p sle-module-confidential-computing/15.6/x86_64</screen>
+    </listitem>
+  <listitem> <para>To enable the Confidential Computing Technical Preview Module, run the command:</para>
+    <screen># suseconnect -p sle-module-confidential-computing/15.6/x86_64
+      Registering system to SUSE Customer Center
+      
+      Updating system details on https://scc.suse.com ...
+      
+      Activating sle-module-confidential-computing 15.6 x86_64 ...
+      Adding service to system ...
+      Installing release package ...
+      
+      Successfully registered system</screen>
+      <para>The confidential compute module is enabled and you can install the packages.</para>
+    </listitem>
+    </itemizedlist>
+  </sect1>
+  <sect1 xml:id="vm-security-verify-setup">
+    <title>Installing Packages and Setting up the Base System</title>
+
+    <para>
+      The confidential compute module provides replacement packages supporting AMD SEV-SNP. To ensure a maximum of compatibility, these packages are based on the code streams from SUSE Linux Enterprise Server.</para>
+<para>The three components that need to be replaced are:</para>
+<itemizedlist>
+  <listitem>
+    <para>The Linux kernel</para>
+  </listitem>
+<listitem>
+  <para>QEMU Virtual Machine Monitor</para>
+</listitem>
+<listitem>
+  <para>&libvirt; framework</para>
+</listitem>
+</itemizedlist>
+<procedure>
+  <step>
+    <para>To install the replacement packages, run the command:</para>
+    <screen># sudo zypper install coco:kernel-coco coco:qemu coco:libvirt
+      <!-- TO DO: Replace with the actual command.-->
+    </screen>
+    <para>After replacing the packages, you must set up the system with a configuration change to make the AMD SEV-SNP feature ready to use. The IOMMU on the host side must be configured in non-passthrough mode. This is required to prevent peripheral devices from writing to memory which belongs to an encrypted guest and destroy its data integrity. The default IOMMU configuration in SUSE Linux Enterprise Server is <literal>passthrough</literal> mode.</para>
```

Comment:
```suggestion
    <para>After replacing the packages, you must set up the system with a configuration change to make the AMD SEV-SNP feature ready to use. The IOMMU on the host side must be configured in non-passthrough mode. This is required to prevent peripheral devices from writing to memory which belongs to an encrypted guest and destroy its data integrity. The default IOMMU configuration in &productname; is <literal>passthrough</literal> mode.</para>
```